benchto-driver/src/main/java/com/teradata/benchto/driver/BenchmarkProperties.java

```diff
@@ -97,12 +96,12 @@ public String getBenchmarksDir()
 
     public Path benchmarksFilesPath()
     {
-        return asPath(getBenchmarksDir());
+        return Paths.get(getBenchmarksDir());
```

Comment:
I guess you could just declare the `benchmarksDir` field as `Path`. (I didn't check though)